applications/nrf5340_audio/child_image/b0n.conf

```diff
@@ -0,0 +1,4 @@
+# SPDX-License-Identifier: Apache-2.0
+CONFIG_RPMSG_SERVICE=n
```

Comment:
Nitpick: Add a newline between the licence and the configs. This should be done for all .conf files in this PR

Reply:
Re-format all config file as advised.